### PR TITLE
Add BREAKING.md updates from recent changes

### PR DESCRIPTION
**Pull request by skylerjokiel**
Adding a collection of recent API breaking changes from the 3P space to Breaking.md.

### CODE REVIEW

File: BREAKING.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 - [Property removed from IFluidDataStoreContext](#Property-removed-from-IFluidDataStoreContext)
 - [Changes to IFluidDataStoreFactory](#Changes-to-IFluidDataStoreFactory)
 - [FlushMode enum values renamed](#FlushMode-enum-values-renamed)
+- [name removed from ContainerSchema](#name-removed-from-ContainerSchema)
+- [Anonymous return types for container calls in client packages](#Anonymous-return-types-for-container-calls-in-client-packages)
+- [createContainer and getContainer response objects properties renamed](#createContainer-and-getContainer-response-objects-properties-renamed)
+- [tinylicious and azure clients createContainer now detached](#tinylicious-and-azure-clients-createContainer-now-detached)
+- [container id is returned from new attach() and not exposed on the container](#container-id-is-returned-from-new-attach()-and-not-exposed-on-the-container)
```
Comment (heliocliu) [inline BREAKING.md:9]: nit: this link doesn't seem to work, probably the `()` needs to be removed?

File: BREAKING.md [hunk 2]
```diff
@@ -14,6 +19,73 @@
 - `FlushMode.Manual` to `FlushMode.TurnBased`
 - `FlushMode.Automatic` to `FlushMode.Immediate`
 
+### `name` removed from ContainerSchema
+The `name` property on the ContainerSchema was used for multi-container scenarios but has not materialized to be a useful schema property. The feedback has been negative to neutral and the goal is to remove it before it becomes formalized. Support for multi-container scenarios, if any is required, will be addressed as a future change.
```
Comment (heliocliu) [inline BREAKING.md:23]:
```suggestion
The `name` property on the ContainerSchema was used for multi-container scenarios but has not materialized to be a useful schema property. The feedback has been negative to neutral so it is being removed before it becomes formalized. Support for multi-container scenarios, if any is required, will be addressed as a future change.
```

File: BREAKING.md [hunk 3]
```diff
@@ -14,6 +19,73 @@
...
+
+- fluidContainer => container
+- containerServices => services
+
+```javascript
+// old
+const { fluidContainer, containerServices } = client.getContainer;
+
+// new
+const { container, services } = client.getContainer;
```
Comment (heliocliu) [inline BREAKING.md:61]:
```suggestion
// old
const { fluidContainer, containerServices } = client.getContainer(...);

// new
const { container, services } = client.getContainer(...);
```
nitty nitty

File: BREAKING.md (hunk 2 above)
Comment (heliocliu) [reply BREAKING.md:23]: Actually this is the same note as `### TinyliciousClient and AzureClient container API changed` below right?  So it's not a 0.47 change?

**Review by heliocliu — APPROVED** [review]

File: BREAKING.md (hunk 1 above)
Comment (skylerjokiel) [reply BREAKING.md]: Hmm interesting. I actually checked this and it works in vscode. Removing them worked too so I did that but yea. Nice catch :)

**Pull request merged**